src/react-valence-ui-iframe.js

```diff
@@ -44,6 +48,26 @@ var ResizingIframe = React.createClass({
 			}
 		}
 	},
+	updateNavbarStyle: function() {
+		// Hide the navbar and minibar from within the iframe if it is rendered
+		var iframe = React.findDOMNode(this.refs.iframe);
+		if (iframe) {
+			// Can only manipulate the iframe if it is not cross domain
+			if (!ResizeCallbackMaker.crossDomain(iframe)) {
```

Comment:
If you set a `sameDomain` state value in the handleOnLoad (using the `security` value in the result object), we can avoid calling this a bunch